### DISCUSSION TITLE
What does Hubs Cloud do when room size limit is reached?

### DISCUSSION THREAD

**Discussion by github-user-77** [mozilla/hubs#3272] [Hubs Cloud Questions] [answered]
Hi All, What does Hubs Cloud do when room size limit is reached? Are there docs I can consult on this – I'm trying to figure out how to accommodate a lot of people turning up for a launch event.

**Comment by github-user-77** [comment, marked as answer]
From empirical testing, what happens once the room limit is reached, is new users can't enter the room, they get a dialogue box with a button that says "Watch from Lobby, Room is Full". Clicking that button dismisses it and the user can then see the room and send messages to the room, without entering the room.

It would be great to be able to direct these users to an overflow room automatically, where the overflow room has a video feed from the full room. Users in the overflow room could at least interact with each other as audience members while watching the feed. But I'm not sure how that could be achieved

**Reply by robinkwilson** [reply]
Make a feature request for it here!: https://github.com/mozilla/hubs/issues
